dbfit-java/core/src/main/java/dbfit/api/DBEnvironment.java

```diff
@@ -189,4 +189,6 @@ void connectUsingFile(String filePath) throws SQLException, IOException,
             FileNotFoundException;
 
     DbStoredProcedureCall newStoredProcedureCall(String name, DbParameterAccessor[] accessors);
+
+    String buildFunctionCall(String procOrFuncName, int numberOfParameters);
```

Comment:
> But what about just extending DbStoredProcedureCall? I think this way we can avoid the need for tweaking the environment API. So far this has been the way to customize behavior e.g. for Oracle.

It seems by doing that that we are hiding, or at least not making explicit, the requirements/contract that an environment vendor must fulfil. We have our implementation and we can get away with changes under the covers. 

What would you think about establishing a minimal start at a vendor interface?